addons/hr_timesheet/models/hr_timesheet.py

```diff
@@ -124,14 +124,18 @@ def fields_view_get(self, view_id=None, view_type='form', toolbar=False, submenu
         return result
 
     @api.model
-    def _apply_timesheet_label(self, view_arch, view_type='form'):
+    def _apply_timesheet_label(self, view_arch, view_type='form', postfix=''):
         doc = etree.XML(view_arch)
         encoding_uom = self.env.company.timesheet_encode_uom_id
         # Here, we select only the unit_amount field having no string set to give priority to
         # custom inheretied view stored in database. Even if normally, no xpath can be done on
         # 'string' attribute.
         for node in doc.xpath("//field[@name='unit_amount'][@widget='timesheet_uom'][not(@string)]"):
-            node.set('string', _('Duration (%s)') % (re.sub(r'[\(\)]', '', encoding_uom.name or '')))
+            if postfix:
+                label = _('%s %s') % (re.sub(r'[\(\)]', '', encoding_uom.name or ''), postfix)
+            else:
+                label = _('Duration (%s)') % (re.sub(r'[\(\)]', '', encoding_uom.name or ''))
+            node.set('string', label)
```

Comment:
I am not sure we have to modify this method, see my comment in your enterprise branch. 🙂